src/commands/run.js

```diff
@@ -80,6 +80,12 @@ class RunCommand extends ApifyCommand {
         // Increases default size of headers. The original limit was 80kb, but from node 10+ they decided to lower it to 8kb.
         // However they did not think about all the sites there with large headers,
         // so we put back the old limit of 80kb, which seems to work just fine.
+        if (semver.gte(process.versions.node, semver.minVersion(SUPPORTED_NODEJS_VERSION))) {
+            env.NODE_OPTIONS = env.NODE_OPTIONS ? `${env.NODE_OPTIONS} --max-http-header-size=80000` : '--max-http-header-size=80000';
+        } else {
+            warning('You are running an version of nodejs, which is not supported. '
```

Comment:
`You are running Node.js version XXX, which is no longer supported. Please upgrade to Node.js version XXX or later`